interfaces/builtin/desktop.go

```diff
@@ -160,6 +160,63 @@ dbus (send)
 # Allow access to the ICC profiles in the home directory to
 # be referred to from colord
 owner @{HOME}/.local/share/icc r,
+
+
+# Allow to send updates to the desktop session about on going jobs
+# (for progress display in the task list)
+dbus (send)
+    bus=session
+    interface=com.canonical.Unity.LauncherEntry
+    member=Update
+    peer=(label=unconfined),
+
+# Allow to send updates to the desktop session about on going jobs
+# (for KDE Plasma specific details)
+dbus (send)
+    bus=session
+    interface=org.kde.JobViewServer*
```

Comment:
Do we need the wildcard?

Reply:
Yes, because there are several version of the interface, we basically have `org.kde.JobViewServer` and `org.kde.JobViewServerV2`. There might be more in the future and you probably don't want to track them here.